app/src/main/java/com/onboarding/wheatherapp/mvp/presenter/WeatherAppPresenter.kt

```diff
@@ -1,14 +1,23 @@
 package com.onboarding.wheatherapp.mvp.presenter
 
 import com.onboarding.wheatherapp.mvp.WeatherAppContract
+import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers
+import io.reactivex.rxjava3.schedulers.Schedulers
 
 class WeatherAppPresenter(
     private val model: WeatherAppContract.Model,
     private val view: WeatherAppContract.View
 ) : WeatherAppContract.Presenter {
 
+
     override fun getFiveDays() {
-        model.getData()
-        view.showData()
+        model.getData(CITY).subscribeOn(Schedulers.io()).observeOn(AndroidSchedulers.mainThread())
+            .subscribe { data ->
+                view.showData(data)
+            }
```

Comment:
this can fit on a single line

Reply:
each fun of the builder should be on separate lines in case they all don't fit on a single one

Reply:
Solved!